src/site/_includes/layout.njk

```diff
@@ -18,6 +18,8 @@
       href="//fonts.googleapis.com/css?family=Google+Sans:400,500|Roboto:400,400italic,500,500italic|Roboto+Condensed:400,700|Roboto+Mono:400,500|Material+Icons"
     />
     <link rel="stylesheet" href="/app.css" />
+    {# Module code is loaded deferred (the "asynchronous" case in the polyfill docs) but we only load other module code anyway. #}
+    <script type="module" src="/lib/webcomponents/webcomponents-loader.js"></script>
```

Comment:
The docs seem to indicate that the polyfills themselves will be loaded async. Is webcomponents-loader just a boostrap that then script-appends the polyfills? If so, we might have to use their `waitFor` API, which feels like a bummer but maybe we can just use it to boostrap app.js or something.

Reply:
Ugh, yeah. I think I failed to brain yesterday, of course we now have to `waitFor` this to be ready.

I'll have a think and update this soon.

Reply:
I feel dirty (?) but I've inlined the loader now.

Reply:
I guess another option would be to create a bootstrap.js that dynamically imports app.js. I think I might prefer this approach as we might do more in bootstrap.js as we're planning to break up app.js and be a bit smarter about what we load and when.

Reply:
I've done this now. I like it. I can also preload `app.js` in a `<link>` tag, which I think makes me happy.